src/utils/read-swagger.ts

```diff
@@ -0,0 +1,20 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { SwaggerDefinition } from '../types';
+
+export function readSwagger(dir: string): SwaggerDefinition {
+    const fileNames = fs.readdirSync(dir);
```

Comment:
We should at least filter files by .json extension, so that presence of some other file (example .DS_Store) does not result in error. Also - please see my other comment regarding purpose of `--swaggerDir` - that would make this piece of code redudant